### PR TITLE
M2-4378   [Admin Panel] Part of entered text is randomly removed after some text was deleted or symbols were added

### PR DESCRIPTION
**Pull request by muhurov**

### CODE REVIEW

File: src/shared/components/FormComponents/EditorController/EditorController.tsx [hunk 1]
```diff
@@ -46,57 +33,24 @@ export const EditorController = <T extends FieldValues>({
...
-                  />
-                )}
-              </StyledFlexSpaceBetween>
-              {isLoading && <Spinner uiType={SpinnerUiType.Secondary} />}
-            </StyledFlexColumn>
-          );
-        }}
+        render={({ field: { onChange, value }, fieldState: { error } }) => (
+          <Editor
+            editorId={editorId}
```
Comment (IhorNesterenko99) [inline src/shared/components/FormComponents/EditorController/EditorController.tsx:38]: could you pls pass some props as ...props, without detailing them?

File: src/shared/components/FormComponents/EditorController/EditorController.tsx [hunk 2]
```diff
@@ -46,57 +33,24 @@ export const EditorController = <T extends FieldValues>({
...
+            editorRef={editorRef}
+            value={value}
+            onChange={onChange}
+            onInsert={onInsert}
+            onFileExceeded={setFileSizeExceeded}
+            onIncorrectFileFormat={setIncorrectFileFormat}
+            onLoading={setIsLoading}
```
Comment (IhorNesterenko99) [inline src/shared/components/FormComponents/EditorController/EditorController.tsx]: maybe move isLoading and setIsLoading to the Editor component?

File: src/shared/components/FormComponents/EditorController/Editor/Editor.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  disabled,
+  isLoading,
+  withDebounce,
+  'data-testid': dataTestid,
+}: EditorProps) => {
+  const { t } = useTranslation('app');
+  const [inputValue, setInputValue] = useState(value ?? '');
+
+  const handleDebouncedChange = useCallback(
+    debounce((v) => onChange(v), CHANGE_DEBOUNCE_VALUE),
```
Comment (IhorNesterenko99) [inline src/shared/components/FormComponents/EditorController/Editor/Editor.tsx]: pls change name "v" to more understandable

**Review by alex-i-e — APPROVED** [review]

File: src/shared/components/FormComponents/EditorController/EditorController.tsx (hunk 2 above)
Comment (muhurov) [reply src/shared/components/FormComponents/EditorController/EditorController.tsx]: done

File: src/shared/components/FormComponents/EditorController/Editor/Editor.tsx (hunk 3 above)
Comment (muhurov) [reply src/shared/components/FormComponents/EditorController/Editor/Editor.tsx]: done

**Review by nmialik — APPROVED** [review]

**Review by IhorNesterenko99 — APPROVED** [review]

**Pull request merged**